geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/DtgAgeOffTest.scala

```diff
@@ -90,4 +99,15 @@ class DtgAgeOffTest extends Specification with TestWithFeatureType {
       query(sysDs) must haveSize(0)
     }
   }
+
+  // Scans all GeoMesa Accumulo tables directly and verifies the number of records that the `root` user can see.
+  private def scanDirect(expected: Int) = {
+    val conn: Connector = MiniCluster.cluster.getConnector(MiniCluster.Users.root.name, MiniCluster.Users.root.password)
+    conn.tableOperations().list().asScala.filter(t => t.contains("DtgAgeOffTest_DtgAgeOffTest")).forall { tableName =>
+      val scanner = conn.createScanner(tableName, MiniCluster.Users.root.auths)
+      val count = scanner.asScala.size
+      scanner.close()
```

Comment:
@elahrvivaz Does this look reasonable or do we have a more clever way to close things?

Reply:
good enough for a unit test. normally you'd want a try/finally, or the gm WithClose sugar.